### PR TITLE
[BEAM-14356] fixing a postcommit test

### PR DESCRIPTION
**Pull request by ahmedabu98**
BigQueryIOJsonIT.java is failing due to a missing pipeline option. This PR is for a small fix to get it running properly again.

### CODE REVIEW

**Comment by ahmedabu98** [comment]
Run PostCommit_Java_DataflowV2

**Comment by ahmedabu98** [comment]
R: @apilloud 

This should fix the failing postcommits.
PreCommits failing for other reasons

**Comment by apilloud** [comment]
Run Java PreCommit

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOJsonIT.java [hunk 1]
```diff
@@ -65,7 +66,13 @@
 public class BigQueryIOJsonIT {
   private static final Logger LOG = LoggerFactory.getLogger(BigQueryIOJsonIT.class);
 
-  @Rule public final transient TestPipeline p = TestPipeline.create();
+  private static PipelineOptions testOptions = TestPipeline.testingPipelineOptions();
+
+  static {
+    testOptions.setTempLocation("gs://bigqueryio-json-it-temp/java-tmp");
```
Comment (apilloud) [inline sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOJsonIT.java:72]: How do you expect this test to run for those who don't have access to `gs://bigqueryio-json-it-temp`? We have a bunch of tests that do something like `options.setTempLocation(options.getTempRoot() + "/java-tmp");` can you do something like that?

Comment (ahmedabu98) [reply sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOJsonIT.java]: Was going to make the bucket available for allUsers, but your suggestion is a lot more reasonable. 
Made the change.

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOJsonIT.java [hunk 2]
```diff
@@ -281,6 +289,10 @@ public void readAndValidateRows(BigQueryIOJsonOptions options) {
       bigqueryIO = bigqueryIO.from(options.getInputTable());
     }
 
+    if (options.getReadMethod() == TypedRead.Method.EXPORT) {
+      options.setTempLocation("gs://bigqueryio-json-it-temp/java-tmp");
```
Comment (apilloud) [inline sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOJsonIT.java:293]: nit: this one too...

**Review by apilloud — APPROVED** [review]

**Comment by ahmedabu98** [comment]
Run Java PreCommit

**Comment by ahmedabu98** [comment]
Run Java PreCommit

**Comment by ahmedabu98** [comment]
R: @apilloud

**Review by apilloud — APPROVED** [review]
LGTM

**Comment by apilloud** [comment]
run java postcommit

**Pull request merged**